test/e2e/app/app.go

```diff
@@ -27,6 +27,23 @@ type Application struct {
 	restoreChunks   [][]byte
 }
 
+type Config struct {
```

Comment:
Could this type and its fields please get some documentation? A reference to some other doc that defines these values would be fine.

Reply:
Yup good idea